docs/source/algorithms/ConvertPeaksWorkspace-v1.rst

```diff
@@ -0,0 +1,56 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm provides bi-directional conversion bewteen regular PeaksWorkspace and LeanElasticPeaksWorkspace.
+
+When casting a PeaksWorkspace into a LeanElasticPeaksWorkspace, information related to detector
+(DetID, BankName, Row, Col) is discarded.
+However, other experiment information are retained in the LeanElasticPeaksWorkspace, including
+run, hkl, goniometer, sample, as well as the corresponding information at the invidual Peak level.
+
+When casting a LeanElasticPeaksWorkspace into a PeaksWorkspace, a donor PeaksWorkspace with proper
+instrument configuration is required.
+Incorrect goniometer setting will lead to a Peak with negative wavelength, which will be excluded from the output
+PeaksWorkspace.
+
+Usage
+-----
+
+1) PeaksWorkspace to LeanElasticPeaksWorkspace
+
+.. code-block:: python
```

Comment:
Would it be hard to make these actual doc-tests instead of just code-bocks?

Reply:
I'd rather keep them as simple code-blocks